docs/gestor-tareas.md

```diff
@@ -0,0 +1,10 @@
+# Gestor de tareas
+
+## Grunt
```

Comment:
Por favor, aplicad lo que os he dicho a todas las herramientas. En *todos* los casos hay que poner los criterios de elección. En *todos* debes elegir una herramienta de acuerdo a esos criterios.

Reply:
Si no habéis modificado la línea comentada pero arreglado lo que se dice, resolved el comentario con una referencia al commit que lo arregla.